docs/clients/fluentd/README.md

```diff
@@ -119,18 +119,20 @@ This plugin automatically adds a `fluentd_thread` label with the name of the buf
 
 ## Docker Image
 
-There is a Docker image `grafana/fluent-plugin-grafana-loki:master` which contains [default configuration files](https://github.com/grafana/loki/tree/master/fluentd/fluent-plugin-grafana-loki/docker/conf). By default, fluentd containers use the configurations but you can also specify your `fluentd.conf` with `FLUENTD_CONF` environment variable.
+There is a Docker image `grafana/fluent-plugin-loki:master` which contains [default configuration files](https://github.com/grafana/loki/tree/master/fluentd/fluent-plugin-loki/docker/conf). By default, fluentd containers use the configurations but you can also specify your `fluentd.conf` with `FLUENTD_CONF` environment variable.
```

Comment:
Is Fluentd a proper name? It is lowercase here and capitalized below. Please capitalize consistently.